### DISCUSSION TITLE
2-12. ⚡ 작업 등대를 만들고, 똑똑하게 시간 관리하기

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#45] [2장 똑똑하게 학습하고 싶어]
# 작업등대: 학습 목표 세분화 및 집중력 강화 실습
작업등대는 학습 목표를 작은 단위로 세분화하여 각 단계를 집중적으로 수행하는 방법입니다. 이를 통해 학습 과정을 더 효과적으로 관리하고, 더 몰입하는 학습 과정을 설계할 수 있습니다.

## 1. 오늘 해야 할 가장 중요한 학습 목표 선정
오늘 가장 중요하게 생각하는 학습 목표를 하나 선정하세요. 이 목표는 하루 안에 달성할 수 있는 구체적이고 실현 가능한 것이어야 합니다.

예: JavaScript의 'this' 키워드에 대한 기본 개념과 사용 방법을 이해하는 것

## 2. 목표를 5~10분 단위의 작은 단계로 나누고 완료 조건을 적어보기
선정한 목표를 5~10분 동안 수행할 수 있는 더 작은 단계로 나누세요. 각 단계는 구체적이고 완료 가능한 작업이어야 합니다. 그리고 각 단계에 대한 명확한 완료 조건을 정의하세요. 이를 통해 각 단계의 성공적인 완료를 명확하게 인식하고 메타인지를 높일 수 있습니다.

단계별 학습 계획 및 완료 조건 예시

- 1단계 (10분): 'this' 기본 개념 읽기
  - 학습 목표: 공식 문서나 신뢰할 수 있는 학습 자료에서 'this'에 대한 기본 설명을 읽고 주요 포인트를 정리
  - 완료 조건: 'this'의 기본 개념에 대한 요약 노트 작성 및 핵심 포인트 3가지 이상 정리

- 2단계 (10분): 예제 코드 분석
  - 학습 목표: 'this'가 사용된 간단한 예제 코드를 찾아 분석
  - 완료 조건: 예제 코드에 대한 분석 노트 작성 및 'this'의 역할과 동작 이해하기

- 3단계 (10분): 실습하기
  - 학습 목표: 직접 간단한 함수를 작성하고 'this'를 사용
  - 완료 조건: 작성한 예제 코드의 실행 결과 확인 및 'this'의 동작 방식에 대한 설명 노트 작성

## 4. 집중 실행 및 단계별 체크
실제로 각 단계를 수행하면서, 완료 조건에 따라 단계별로 체크하고 진행 상황을 기록해보세요. 각 단계를 완료한 후에는 잠시 쉬어가면서 다음 단계를 준비하세요.

## 5. 실습 후기 및 자가 평가
모든 단계를 완료한 후, 오늘의 학습 과정에 대해 회고를 작성해 보세요.

<br>
<br>

# 📝 아래의 템플릿을 복사해서 작성해주세요
```
## 1. 오늘의 학습 목표
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 
  - 학습 목표: 
  - 완료 조건:
- 2단계 (10분): 
  - 학습 목표: 
  - 완료 조건:
- 3단계 (10분): 
  - 학습 목표: 
  - 완료 조건:

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

```

# 🔗 참고 링크
[하루스터디](https://haru-study.com)

**Comment by pitepan3** [comment]


**Reply by imakerjun** [reply]
오늘의 하이라이트인 학습 목표가 '반복문'인것이죠~?

**Comment by D0-HA** [comment]
## 1. 오늘의 학습 목표

호이스팅의 개념과 `var`와 `let`의 차이 이해하기

## 2. 단계별 학습 계획 및 완료 조건

- 1단계 (10분): 호이스팅 개념
  - 학습 목표: 자바스크립트에서 호이스팅의 원리에 대해 이해할 수 있다.
  - 완료 조건: 호이스팅 개념 한 줄 정리
  
- 2단계 (10분): `var`사용시 생기는 오류 예시 알아보기
  - 학습 목표:  `var`를 더이상 쓰지않는 이유를 알 수 있다.
  - 완료 조건: 예시 3가지 작성
  
- 3단계 (10분): `let`의 오류 해결 방법 이해
  - 학습 목표:  TDZ에 대해 이해할 수 있다.
  - 완료 조건: TDZ 동작 원리 한 줄 정리

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by lovemagician** [comment]
## 1. 오늘의 학습 목표
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 자바스크립트 인포 사이트의 switch문 파트를 읽어보자.
  - 학습 목표: switch문에 대해 알아보기
  - 완료 조건: 비유를 하여 설명할 수 있다.
- 2단계 (10분): 예제 코드를 직접 작성해보자.
  - 학습 목표: switch문을 작성 해보기
  - 완료 조건: switch문을 작성할 수 있다.
- 3단계 (10분): 예제 코드를 기준으로 나만의 상황을 대입하여 직접 활용해보자.
  - 학습 목표: switch문이 필요한 상황들을 다양하게 느껴보기.
  - 완료 조건: switch문이 적합한 상황을 떠올리며 switch문을 작성 할 수 있다.

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by godheezzang** [comment]
## 1. 오늘의 학습 목표
JavaScript의 비교 연산자에 대한 기본적인 이해 완료하기

## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 비교 연산자의 개념 익히기
  - 학습 목표: 공식 문서에서 비교 연산자 대한 부분을 발췌하여 읽고 키워드 정리
  - 완료 조건: 단락 별로 2-3줄 요약하여 정리하고 예시 만들어보기
- 2단계 (10분): 예제 코드에서 기본 연산자와 비교 연산자가 어떻게 작동되는지 훑어본다.
  - 학습 목표: 예제 코드에서 '왜' 이런 연산자가 사용되었는지 이해하기
  - 완료 조건: 다른 연산자를 사용한다면 '어떻게' 바꿔 사용할 수 있는지 생각해보고 코드 짜보기
- 3단계 (10분): 비교 연산자 실습
  - 학습 목표: 연산자를 이용한 문제를 만들고 원리 이해하기
  - 완료 조건: 실습 문제를 통해 비교 연산자의 알고리즘을 파악 후 노트 정리

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by nonefic** [comment]
## 1. 오늘의 학습 목표
객체에 대한 개념 이해와 어떤식으로 응용할수있는지 사용해보기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 객체에 대해 어떤 개념인지 알아보기
  - 학습 목표: 강의 듣고 객체가 어떤개념인지 큰흐름으로 이해하기
  - 완료 조건:강의 들은내용으로 바탕으로 요약하기
  - 
- 2단계 (10분): 강의를 듣고 어느정도 이해했으면 공식문서에서 정독
  - 학습 목표: 강의와 교차로 공식문서를 이용하여 개념이해
  - 완료 조건:두 내용을 바탕으로 이해한 내용 노트에 작성
- 3단계 (10분): 응용단계
  - 학습 목표: 객체에 대해 코드로 작성
  - 완료 조건:노트에 작성한 내용을 최대한 안보고 코드를 만져보면서 다시한번 복기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by kihonoluluhawaii** [comment]
## 1. 오늘의 학습 목표
- 함수의 기본개념을 이해하기
- 
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분):  함수의 개념정리
  - 학습 목표:  공식문서를 통해 함수의 원리를 이해하고 개념파악하기 
  - 완료 조건: 열줄 요약해서 정리해보기
- 2단계 (10분): 함수 예제코드 찾아보기
  - 학습 목표: 기본적인 함수 코드가 들어간 예제를 찾아보고 분석하기
  - 완료 조건: 코드를 이해하고 의미를 파악해서 리뷰해보기
- 3단계 (10분): 함수 실습하기
  - 학습 목표: 직접 간단한 함수 코드를 작성해서 실행시켜보기
  - 완료 조건: 직접 만든 코드를 콘솔창에 띄워 원하던 결과가 나오는지 확인하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by g0meee** [comment]
## 1. 오늘의 학습 목표
함수 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 
  - 학습 목표: 함수 공식 문서 읽기
  - 완료 조건: 핵심 키워드나 모르는 것 3가지 이상 정리
- 2단계 (10분): 
  - 학습 목표: 핵심 키워드를 확실히 이해하고, 모르는 부분 공부하기
  - 완료 조건: 모르는 부분 구글링, 예제 찾으면서 공부하기
- 3단계 (10분): 
  - 학습 목표: 실습 문제 풀어보기
  - 완료 조건: 실습 문제를 풀어보면서 내용을 이해하고 헷갈리는 내용 체크하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by gpmjeongeun** [comment]
## 1. 오늘의 학습 목표
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 메서드 개념 읽기 
  - 학습 목표:  모던 자바스크립트 딥 다이브 메서드 읽고 이해하기
  - 완료 조건: 정리해보기
- 2단계 (10분):  예제코드 이해
  - 학습 목표:  예제코드 작성후 이해하기
  - 완료 조건: 이해한 예제코드 정리
- 3단계 (10분): 실습
  - 학습 목표:  
  - 완료 조건:

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by chaeyoungg** [comment]
## 1. 오늘의 학습 목표
자바스크립트 함수 이론 학습 & 실습 진행해보기 

## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 함수에 대한 기본 이론 정독하고 이해해보기
  - 학습 목표: 제공된 교안이나 공식 문서 등을 통해 자바스크립트 함수에 대한 이론적인 부분 학습하기 
  - 완료 조건: 학습한 함수 기본 개념에 대한 요약 정리 블로그 글 발행

- 2단계 (10분): 함수 사용 예제 코드 실습해보기  
  - 학습 목표: 함수 사용 예제 따라서 타이핑하며 실습해보기
  - 완료 조건: 예제 코드 분석하고 함수의 사용법, 동작에 대해 완벽하게 이해하기

- 3단계 (10분): 함수 사용 과제 수행하기
  - 학습 목표: 함수 사용된 과제 교안의 도움 없이 직접 작성해보기 
  - 완료 조건: 교안의 도움 없이 익숙하게 함수 사용

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by noohijimik** [comment]
## 1. 오늘의 학습 목표

모던 자바스크립트 튜토리얼 함수 챕터 학습하고 이해하기


## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (20분): 함수의 정의와 기본 개념 익히기
  - 학습 목표: 함수의 정의를 이해하고 서술 할 수 있도록 해당 개념을 익힘.
  - 완료 조건: 학습 시 선정한 키워드와 해당하는 내용 모두 막힘 없이 서술하며 온전히 이해 함.
  - 
- 2단계 (20분):  1단계에서 뚜렷하게 이해되지 않은 핵심 개념에 해당하는 예제 코드 분석
  - 학습 목표: 예제 코드를 옮겨 적어 주석과 결과 값을 지운 채로 예제 코드의 동작 방식을 분석하고 결과 까지 도출.
  - 완료 조건: 분석한 코드와 도출한 결과 값이 코드를 실행 했을 때 나온 결과 값과 일치.
  - 
- 3단계 (10분): 이해되지 않았던 개념에 대해 다시 한번 되짚은 후, 직접 코드를 작성하여 해당 개념에 대해 숙지
  - 학습 목표: 직접 코드를 작성하되, 예제에서 보여주고자 했던 개념을 작성한 코드에서도 동일하게 녹여낼 것.
  - 완료 조건: 작성한 코드와 예제 코드가 동일하게 작동하며 해당 개념에 대한 설명이 가능.


## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by kyungmim** [comment]
## 1. 오늘의 학습 목표
- 오늘 배운 Javascript 수업 블로그에 정리하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 흐름 이해하기
  - 학습 목표: 수업에 쓰였던 Notion문서 읽어보면서 수업내용 복습하기
  - 완료 조건:  구글에 궁금한점을 정확하게 서치 할 수 있을정도로 읽어보기
- 2단계 (10분): 추가하기
  - 학습 목표: 배운 내용을 복습했다면 구글 서치를 통해 궁금한점 찾아보면서 추가 학습하기
  - 완료 조건: 4-5번의 구글링이 아닌 3번의 구글링으로 궁금한점 찾아내기
- 3단계 (10분): 정리하기
  - 학습 목표: 블로그 작성 큰 틀 잡기
  - 완료 조건: 대제목과 소제목 분류 하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by loeybho** [comment]
## 1. 오늘의 학습 목표:
loop 기본 개념 정복
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분):  while문 기본 익히기
  - 학습 목표: MDN에서 while문 읽고, 드림 코딩 영상 1개 보기
  - 완료 조건: while이 무엇인지 눈에 보이게 개념 정리
- 2단계 (10분): for문 기본 익히기
  - 학습 목표: MDN에서 for문 읽고, 드림코딩 영상 1개 보기
  - 완료 조건: for이 무엇인지 눈에 보이게 개념 정리
- 3단계 (10분): while, for 문 활용하기
  - 학습 목표: while,for 활용하여 예제 코드 작성
  - 완료 조건: while과 for 식은 보지 않고 빈 종이에 쓸 수 있어야 함

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by HaeppyRi** [comment]
## 1. 오늘의 학습 목표

모던자바스크립트 사이트 ‘함수’ 학습하고 이해하기

## 2. 단계별 학습 계획 및 완료 조건

- 1단계 (10분):  ‘function’ 관련 기본 개념 읽기
    - 학습 목표: <자바스크립트 인포> 사이트에서 함수 선언문, 식, 화살표 함수 식 등 ‘function’에 대한 기본 설명을 다시 읽고 중요 키워드 뽑아보기
    - 완료 조건: ‘function’의 기본 개념에 대한 노션 작성 및 중요 키워드 3가지 이상 정리하기
- 2단계 (10분): <자바스크립트 인포> 및 수업 실습 예제 분석하기
    - 학습 목표: ‘function’이 사용된 예제 코드 분석
    - 완료 조건: 모든 예제 풀기, 모르는 문제는 넘기기
- 3단계 (10분): 다시 처음으로 돌아가 개념을 읽고, 해결되지 않은 예제를 왜 풀지 못했는지 분석하기
    - 학습 목표: 해결하지 못한 예제를 해결하기 위해 다시 기본 개념을 익히고, 어느 부분에서 막혔는지 찾아 문제점을 해결한다
    - 완료 조건: 작성한 예제 코드를 어떻게 해결했는지, 어느 부분에서의 지식이 부족했던 것인지 구체적으로 노션에 정리하기

**Comment by macklinkim** [comment]
## 1. 오늘의 학습 목표
강의 수업 잘 따라가기, JavaScript 함수, 객체의 특징 알기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 함수 typeof 연산자의 사용 방법 익히기
  - 학습 목표: typeof의 사용방법, 정의를 스스로 정리해보기.
  - 완료 조건: 강의 페이지 및 도서, mdn의 typeof 용법을 각기 다른 곳에서 읽어보기
- 2단계 (10분): 예제 문제 2~3개 풀기
  - 학습 목표: 예제 문제 풀이를 통한 typeof 키워드 사용에 익숙해지기
  - 완료 조건: 답안지를 읽지 않고 쉬운 난이도의 typeof 문제를 3분 제한시간을 두고 풀기
- 3단계 (10분): 머릿속으로 typeof사용을 숙지할것
  - 학습 목표: typeof 사용이 필요한 순간, 바로 나올 수 있도록 숙지
  - 완료 조건: typeof 사용이 예상되는 순간을 몇가지 정리해서 나누기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by do990118** [comment]
## 1. 오늘의 학습 목표
자바스크립트의 '배열'에 대한 기본 개념을 이해한다

## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (15분): 수업 시간에 받은 학습 자료에서 '배열'에 관한 문서를 찾고 읽어 보기
  - 학습 목표: '배열'의 개념을 이해하고 받아들인다
  - 완료 조건: 질문하고 싶은 부분을 1 개 이상 찾는다
- 2단계 (15분): 실습하기
  - 학습 목표: 이해한 개념을 바탕으로 실습하여 공부한 내용을 확인한다
  - 완료 조건: 학습 자료에 있는 과제를 해결한다

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by pitepan3** [comment]
## 1. 오늘의 학습 목표
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 
  - 학습 목표:  배운 문법 정리, 요약하기
  - 완료 조건:  남들도 보고 이해할수 있도록
- 2단계 (10분): 
  - 학습 목표:  스스로에게 예제 만들어보기
  - 완료 조건:  참고자료 없이 풀어보기
- 3단계 (10분): 
  - 학습 목표:  못풀었다면 구글링해서 풀어보기
  - 완료 조건:  완벽한 문제풀이

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by HuiseonDev** [comment]
1. 오늘의 학습 목표
JavaScript의 개념 익히기

2. 단계별 학습 계획 및 완료 조건
1단계 (10분): JavaScript 문법들의 개념 익히기
학습 목표: 공식문서를 참조하여 내용 정리하기
완료 조건: 예시 코드를 만들어 작성해보기
2단계 (10분): Notion에 정리하기
학습 목표: 내가 이해한 코드의 정의에 대해 명확히 정리
완료 조건: 
3단계 (10분): 코드 실습
학습 목표: 다양한 예시 코드들도 서치 해 보고 유형별로 풀이 해보기
완료 조건: 처음 접한 형식의 코드들도 풀이 가능한 상태
3. 집중 실행 및 단계별 체크
4. 실습 후기 및 자가 평가

**Comment by s0zzang** [comment]
## 1. 오늘의 학습 목표
- 프로토타입 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (20분): 프로토타입 개념 이해
  - 학습 목표: 모던 자바스크립트 딥다이브 프로토타입 읽고 정리
  - 완료 조건: 프로토타입 객체 기본 개념 요약 노트 작성
- 2단계 (20분): 프로토타입 적용 이해
  - 학습 목표: 모던 자바스크립트 딥다이브 프로토타입 예제 분석
  - 완료 조건: 어떻게 적용되는지 파악하기 위해 예제 코드 3개 이상 작성
- 3단계 (20분): 프로토타입 정리
  - 학습 목표: 요약 노트와 코드를 복습하며 프로토타입 개념 정리
  - 완료 조건: 프로토타입 벨로그 작성

**Comment by bedlam3** [comment]
## 1. 오늘의 학습 목표
- 반복문 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 반복문 자료 읽어보기.
  - 학습 목표: 어떤경우에 쓰이는지 어떻게 쓰는지 읽어보기.
  - 완료 조건: 한줄로 요약해서 정리
- 2단계 (10분): 예제 코드로 실습.
  - 학습 목표: 예제 코드를 그대로 타이핑 후 실습 결과 확인, 변수 값도 조금씩 바꿔보기.
  - 완료 조건: 실습결과를 통해 눈으로 확인후 이해하기.
- 3단계 (10분): 간단한 문제 풀어보기.
  - 학습 목표: sum을 구하는 코드 만들어보기.
  - 완료 조건: 자신이 원하는 출력값이 나오면 성공

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by aksendev** [comment]
## 1. 오늘의 학습 목표
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분):  함수란 개념에 대해 알기
  - 학습 목표: 공식문서에서 찾아봐서 읽기
  - 완료 조건: 함수선언문과 표현식의 개념이해
- 2단계 (10분): 예제를 통한 심화 이해
  - 학습 목표: 관련된 예제코드를 찾아보기
  - 완료 조건: 해당 예제코드 따라 작성해보며 이해하기
- 3단계 (10분): 실제 코드 작성해보기
  - 학습 목표: 직접 작성해보기
  - 완료 조건: 원하는 결과가 나오는지, 다른사람에게 설명할 수 있는지 확인하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by JWJung-99** [comment]
## 1. 오늘의 학습 목표
(오늘은 JavaScript보다 기존에 공부하던 CSS에 더 집중할 예정)
HTML input과 form에 대해 복습하고 회원가입 폼 만들어보기

## 2. 단계별 학습 계획 및 완료 조건

- 1단계 (10분): 'input'에 대해 이해하기
  - 학습 목표: 기본적인 text 타입의 input과 select, checkbox에 대한 공식 문서를 읽고 사용법을 이해하기
  - 완료 조건: property들을 파악하여 중요한 포인트들을 2가지 이상 정리하기

- 2단계 (10분): 실제 사용되는 예시들을 분석하기
  - 학습 목표: 회원가입 폼이 실제로 어떻게 구성되어 있는지 개발자 도구를 이용하여 분석하기
  - 완료 조건: 3개 이상의 사이트를 분석하여 form의 구성을 파악하고 공통적인 특징이 있는지 확인하기

- 3단계 (10분): 직접 코드 작성해보고 더 발전시키기
  - 학습 목표: 공부한 내용을 바탕으로 form을 작성하고, 조건을 충족하지 않았을 때 핸들링하는 방법에 대해 구글링 및 공식 문서 확인하기
  - 완료 조건: 디자인 시안에 따라 form을 완성하고, password input을 조건에 따라 컨트롤 해 보기
 
## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by sasakieiji** [comment]
## 1. 오늘의 학습 목표
오늘 배운 자바스크립트 내용 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 수업내용 잘 이해했는지 파악하고 이해가 안되는 부분 명확히 파악하기
  - 학습 목표: 수업시간 필기 내용 보며 이해안되는 부분 명확히 인지하기
  - 완료 조건: 이해 안되는 부분을 명확히 나누어 목록 작성하기
- 2단계 (10분): 이해 안되는 부분을 찾아보기
  - 학습 목표: 목록을 구글링이나 AI한테 물어보기
  - 완료 조건: 모든 목록에 대한 설명이 작성이 되어 있어야함
- 3단계 (10분): 찾아본 내용을 기반으로 수업내용이 잘이해되는지 확인하기
  - 학습 목표: 수업내용을 진정으로 이해했는지 확인하기
  - 완료 조건: 수업내용을 자신만의 언어로 정리하여 알기 쉽게 작성해보기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by De-Chip** [comment]
## 1. 오늘의 학습 목표
자바스크립트의 기본 개념 복습하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 기본 개념 습득하기
  - 학습 목표: 기본 개념에 대하여 이론적인 부분 흡수하기
  - 완료 조건: 선택한 개념의 노션 문서를 읽고, 내 노션에 정리해서 올리기
- 2단계 (10분): 코드로 살펴보기
  - 학습 목표: 실습 코드를 통해 이해도를 높이고 활용법 깨닫기
  - 완료 조건: 노션에 같이 제공된 실습 코드를 따라서 작성 완료하기
- 3단계 (10분): 코드 이해하기&반복 학습
  - 학습 목표: 단순히 코드를 따라치는 것이 아닌, 다양하게 변화를 준 코드 작성하기
  - 완료 조건: 실습 코드를 보지 않고, 공부했던 개념을 활용하여 새로운 코드 작성하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by sozzi0** [comment]
## 1. 오늘의 학습 목표
자바스크립트 함수 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 자바스크립트 인포 문서 읽기
  - 학습 목표: 함수 파트의 예습
  - 완료 조건: 중요한 키워드 기억하기
- 2단계 (10분): 
  - 학습 목표: 
  - 완료 조건:
- 3단계 (10분): 
  - 학습 목표: 
  - 완료 조건:

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by Simhoyoung** [comment]
## 1. 오늘의 학습 목표

   자바스크립트 함수에 대한 이해.

## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 코어 JS 강의 내용 중 함수 파트 내용 읽기
  - 학습 목표: 코어 JS 학습 자료에서 함수 파트를 읽고 내용을 정리한다
  - 완료 조건: 함수에 대한 핵심 포인트 5가지 이상 정리
- 2단계 (10분): 실습 코드 분석
  - 학습 목표: 학습 자료에 첨부된 실습 코드 부분을 복습하며 분석
  - 완료 조건: 실습 코드에 대한 분석으로 이해력 끌어올리기
- 3단계 (10분): 응용 예제
  - 학습 목표: 간단한 응용 예제를 찾아서 작성
  - 완료 조건: 응용 예제를 작성해보고 관련 결과를 찾아보며 비교하고 이해하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by C5D2** [comment]
## 1. 오늘의 학습 목표
if문 이해한 내용 글로 설명해보기

## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): if문 개념 정리하기
 - 학습 목표: if문 관련 내용 mdn 문서 및 정리된 내용 서치해서 읽기
 - 완료 조건: if문에 대해 읽었던 내용 핵심 내용 3~4개로 나누어서 정리
- 2단계 (10분): if문 예제 코드 분석하기
 - 학습목표: if문이 사용된 예제 코드를 2개 찾아 분석
 - 완료조건: 예제 코드 분석 후 분석한 내용 글로 정리하기
- 3단계 (10분): 실습 문제 풀어보기
 - 학습목표: if문 관련 문제 한 개 찾아서 풀어보기
 - 완료조건: 풀었던 내용과 문제 해답 비교하고 정리하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by NAM9184** [comment]
## 1. 오늘의 학습 목표
자바스크립트 내장객체의 기초를 학습해보자 
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분):   내장객체가 무엇인지를 정독하는 시간
  - 학습 목표:  mdn을 통해 내장겍체가 무엇인지 전체적인 틀을 확인한다.
  - 완료 조건:  지금까지 수업에서 활용했던 객체를 확인
- 2단계 (10분):  지금까지 배웠던 내장객체에 대해서 확인해보는 시간
  - 학습 목표:  객체별 특징과 설명을 정독하고 짧게 활용해보자
  - 완료 조건: 활용에 있어서 오류가 나지않고 작성된다면 OK
- 3단계 (10분):  오류개체는 무엇인지 알아보자
  - 학습 목표:  오류객체가 왜 생성되는지 이유를 정독한다.
  - 완료 조건:  오류의 사례를 구글링해보며 해결법을 들여다보

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by danggin** [comment]
## 1. 오늘의 학습 목표
var, let, conset 키워드로 선언한 변수의 차이점 이해하기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 개념 학습
  - 학습 목표: 개념서를 읽고 var, let, conset 키워드로 선언한 변수에 관련된 요점 정리
  - 완료 조건: 각 변수마다 2개 이상의 요점 정리하기
- 2단계 (10분): 예제 분석하기
  - 학습 목표: 각 변수의 차이점을 알 수 있는 예제 코드 찾아 분석하기
  - 완료 조건: 예제 코드를 분석하고 실행 결과에 대한 해설 작성해보기
- 3단계 (10분): 실습해보기
  - 학습 목표: 간단한 예제를 직접 코딩해보고 실제로 실행되는 결과로 var, let, const 변수의 차이점 알아보기
  - 완료 조건: 실습하면서 예상되는 실행 결과와 코드 실행 결과를 비교해보며 차이점을 잘 학습했는지 확인하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가

**Comment by hyedanfe** [comment]
## 1. 오늘의 학습 목표
모던 JavaScript 튜토리얼에서 함수 부분 설명 읽고 과제 풀어보기
## 2. 단계별 학습 계획 및 완료 조건
- 1단계 (10분): 함수 기본 개념 학습하기 
  - 학습 목표: 모던 JS 튜토리얼에서 함수 기본 개념 읽기
  - 완료 조건: 핵심 키워드 3개 이상 뽑아 그걸 중심으로 간단하게 정리해보기
- 2단계 (10분): 함수 기본 원리 및 작성법 익히기
  - 학습 목표: 튜토리얼 예시 코드 직접 작성해보기
  - 완료 조건: 예시 코드 직접 작성 후 작성법 및 원리에 대해 정리해보기
- 3단계 (10분): 과제 풀이를 통해 실습해보기
  - 학습 목표: 튜토리얼 과제 풀어보기
  - 완료 조건: vscode에 직접 작성해보며 과제를 풀어보고 헷갈리는 부분이나 틀린 부분 정리하기

## 3. 집중 실행 및 단계별 체크
## 4. 실습 후기 및 자가 평가
